Changes for the globus issues (#40)
igsr/gca_1000genomes_website: _faq/what-tools-can-i-use-to-download-igsr-data.md

The 1000 Genomes FTP site is available as an end point in the [Globus Online system](https://www.globus.org/).  In order to access the data you need to sign up for an account with Globus via their [signup page](https://app.globus.org/). You must also install the [Globus Connect Personal software](https://www.globus.org/globus-connect-personal) and setup a personal endpoint to download the data too.

> The 1000 Genomes data is hosted at the EMBL-EBI end point called "EMBL-EBI Public Data". Data from our FTP site can then be found under the 1000g directory within the EMBL-EBI Public Data endpoint. The endpoint can be found [here](https://app.globus.org/file-manager?origin_id=47772002-3e5b-4fd3-b97c-18cee38d6df2&origin_path=%2F1000g%2Fftp%2F) 

Reviewer: Is it better to say collection instead of endpoint? 
Docs: https://docs.globus.org/cli/collections_vs_endpoints/#:~:text=The%20Endpoint%20is%20used%20for%20some%20operations%20like%20collection%20listing,management%20capabilities%20and%20data%20transfers.

```suggestion
The 1000 Genomes data is hosted as EMBL-EBI collection called "EMBL-EBI Public Data". Data from our FTP site can then be found under the 1000g directory within the EMBL-EBI Public Data collection. The collection can be found [here](https://app.globus.org/file-manager?origin_id=47772002-3e5b-4fd3-b97c-18cee38d6df2&origin_path=%2F1000g%2Fftp%2F) 

```
